packages/perseus/src/components/svg-image.tsx

```diff
@@ -180,6 +180,16 @@ type Props = {
     setAssetStatus: (assetKey: string, loaded: boolean) => void;
 };
 
+type DefaultProps = {
```

Comment:
Just moved this up to be a named type (our convention in this repo) as opposed to an anonymous, inline type below.